nuxeo-runtime/nuxeo-runtime-mongodb/src/test/java/org/nuxeo/runtime/mongodb/MongoDBFeature.java

```diff
@@ -74,9 +74,11 @@ public void start(FeaturesRunner runner) {
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
-        // finally clear the DB
-        try (MongoClient client = MongoDBConnectionHelper.newMongoClient(server)) {
-            MongoDBConnectionHelper.getDatabase(client, dbname).drop();
+        if (!IgnoreNoMongoDB.test()) {
```

Comment:
perhapse i am missing something, but if this feature (the `MongoDBFeature` one)is ignored due
`@ConditionalIgnoreRule.Ignore(condition = IgnoreNoMongoDB.class, cause = "Needs a MongoDB server!")`
then there is no DB created and then no cleanup is needed 
from what's i understand if the feature is ignored the `start method` is not called and then no DB is created. if it's the case why we add this 2nd control on `!IgnoreNoMongoDB.test()` clean up the DB

Reply:
`start` method is called whenever the feature triggers the ignore or not. 

This is because we check the ignore just before each `@Before`s evaluation because we need Runtime started in some cases.

Our features weren't really designed to handle such class level cases. Current change is a workaround. We will probably review our features mechanism if we migrate to junit 5.

Reply:
i understand, thanks